### PR TITLE
Some modifications in Tutorials on visualization

### PR DESCRIPTION
**Pull request by xueminchi**
Hi @maxspahn ,

In this PR, I did some changes as below:
1): I add some plot tools and apply them (plotTrajtory, plotMultiTrajectory, plotEnergy, etc.)
2): Fix some minor warnings (Circle.patch inputs should be tuple), minor changes to figure size and obstacles.
3): Delete some unnecessary repretive comments.
4): Reformat the code style to a unified style.

The advantage is that when adding new tutorial examples, we don't need to code repretitive plot functions. If we want to have a unified visualization, we can directly modified in 'utils_tutorial' and automatically apply to all tutorials.

Best,
@xueminchi

### CODE REVIEW

File: tutorial/utlis_tutorial.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    ax.set_xlim([-4, 4])
+    ax.set_ylim([-4, 4])
+    ax.plot(x, y, linewidth=1)
+    if ani:
+        (line,) = ax.plot(x, y, color="k", linewidth=1)
+        (point,) = ax.plot(x, y, "r*", markersize=10)
+        return x, y, line, point
+
+
+def plotPolarTraj(sol, ax, fig):
```
Comment (maxspahn) [inline tutorial/utlis_tutorial.py:49]: Why do you use abbrevations here?

File: tutorial/utlis_tutorial.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        ax.plot(t, energies)
+        ax.axis("equal")
+    elif name is not None:
+        ax.plot(t, energies, label=name)
+        ax.legend(loc="upper right")
+    else:
+        ax.plot(t, energies)
+
+
+def plotVelo(velo, ax, fig, name="", equal=None):
```
Comment (maxspahn) [inline tutorial/utlis_tutorial.py:30]: Could you respect the snake variable convention here?
You can check code style by invoking `pylint tutorial/utlis_tutorial.py` from the root of this repository.

**Review by maxspahn — CHANGES_REQUESTED** [review]
@xueminchi 

Thanks for your effort. Creating this plotting function is very useful and improves readibility. 
Why the name `utlis` instead of `utils`?
Can you change the function names to confirm the snake convention?

Best,
@maxspahn

File: tutorial/utlis_tutorial.py (hunk 2 above)
Comment (xueminchi) [reply tutorial/utlis_tutorial.py]: Thanks! I reformat the codes according to the pylint instruction.

**Review by maxspahn — APPROVED** [review]
@xueminchi It looks great. Looking forward to future PR's.

**Pull request merged**